pkg/functions/deploy.go

```diff
@@ -73,6 +73,45 @@ func RunDeploy(w io.Writer, sdk *SdkStatus) error {
 			if err != nil {
 				return err
 			}
+		} else if step.Build.Builder != "" {
+			if step.Build.Destination == "" {
+				return fmt.Errorf("Please provide a destination for the build image.")
+			}
+			if step.Build.MainPath == "" {
+				return fmt.Errorf("Please provide the relative path to the main file.")
+			}
+			packCommand := fmt.Sprintf("pack build %s --builder %s --env GOOGLE_BUILDABLE=%s --publish", step.Build.Destination, step.Build.Builder, step.Build.MainPath)
```

Comment:
what's the GOOGLE_BUILDABLE env var for?

Reply:
It's used by the google buildpacks as a pointer to the main file. I'll rev on this and remove that soon by adding a buildpack to our builder that searches for the main file and sets the env var so we don't have to pass it in.